test/org/opensolaris/opengrok/web/PageConfigTest.java

```diff
@@ -64,7 +64,10 @@ public static void setUpClass() throws Exception {
         repository = new TestRepository();
         repository.create(
                 HistoryGuru.class.getResourceAsStream("repositories.zip"));
-        RuntimeEnvironment.getInstance().setRepositories(repository.getSourceRoot());
+        
+        RuntimeEnvironment env = RuntimeEnvironment.getInstance();
+        env.setHistoryEnabled(true);
```

Comment:
Since `RuntimeEnvironment` is shared between all runs of tests, usually I have noticed a reversion of original settings — e.g. `env.setHistoryEnabled(originalIsHistoryEnabled)` in `@AfterClass`. I see tests passing but wonder if it should be done anyway for correctness.

Reply:
Yeah, could do that. However, if all the test classes ran in the same JVM instance I think we would have failures all over the place.